### PR TITLE
Fix and cleanup views for accepting invitations

### PR DESCRIPTION
**Pull request by rowanseymour**
Includes some UI tweaks:

Captura de pantalla 2023-10-25 a la(s) 11 19 03

becomes:

Captura de pantalla 2023-10-25 a la(s) 11 18 16

and 

Captura de pantalla 2023-10-25 a la(s) 13 31 43

becomes

Captura de pantalla 2023-10-25 a la(s) 13 30 37

and 

Captura de pantalla 2023-10-25 a la(s) 13 48 57

becomes

Captura de pantalla 2023-10-25 a la(s) 14 02 06

### CODE REVIEW

File: temba/orgs/views.py [hunk 1]
```diff
@@ -1125,9 +1151,9 @@ class OrgCRUDL(SmartCRUDL):
         "edit",
         "edit_sub_org",
         "join",
+        "join_signup",
         "join_accept",
         "grant",
-        "create_login",
```
Comment (rowanseymour) [inline temba/orgs/views.py:1130]: renamed for clarity

File: templates/orgs/org_join.html [hunk 2]
```diff
@@ -1,41 +1,44 @@
...
-                         password
-                         id="id_password">
+                         label="{% trans "Password" %}"
+                         placeholder="{% trans "Your password" %}"
+                         password>
         </temba-textinput>
       </div>
     </fieldset>
     {% csrf_token %}
+    <input type="hidden" name="username" value="{{ invitation.email }}">
```
Comment (rowanseymour) [inline templates/orgs/org_join.html:41]: I think when temba-textinput is disabled then the field doesn't actually submit so this redirection to the login page was broken

File: templates/orgs/org_join_signup.html [hunk 3]
```diff
@@ -31,14 +37,3 @@
...
-{% block extra-script %}
-  {{ block.super }}
-  <script type="text/javascript">
-    $(function() {
-      $(".formax-icon, .formax-summary").on('click', function() {
-        $(this).parents(".formax-section").toggleClass('open');
-        $(this).parents(".formax-section").siblings('formax-section').removeClass('open');
-      });
-    });
-  </script>
```
Comment (rowanseymour) [inline templates/orgs/org_join_signup.html:43]: eh?

File: temba/orgs/tests.py [hunk 4]
```diff
@@ -2662,6 +2512,131 @@ def test_workspace(self):
...
+
+        response = self.client.get(join_url)
+        self.assertRedirect(response, join_accept_url)
+
+        # but only if they're the currently logged in user
+        self.login(self.admin)
+
+        response = self.client.get(join_url)
+        self.assertContains(response, "Sign in to join the <b>Nyaruka</b> workspace")
+        self.assertContains(response, f"/users/login/?next={join_accept_url}")
```
Comment (norkans7) [inline temba/orgs/tests.py:2547]: I think this should redirect to `join_url` instead and the view will go to `join_accept_url` in pre_process

Comment (norkans7) [reply temba/orgs/tests.py:2547]: I see the logout line was moved so this is similar too

**Review by norkans7 — APPROVED** [review]

**Pull request merged**